src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/SuggestionHandlers/DottedNameNodeSuggestionHandler.cs

```diff
@@ -177,7 +177,7 @@ private static bool TryGetNamespaceFunctions(TexlNode node, TexlBinding binding,
                 Contracts.AssertValid(firstNameInfo.Name);
 
                 var namespacePath = new DPath().Append(firstNameInfo.Name);
-                functions = binding.NameResolver.LookupFunctionsInNamespace(namespacePath);
+                functions = binding.NameResolver.LookupFunctionsInNamespace(namespacePath).Where(f => f is not ISupportsDeprecatedFunctions sdf || !sdf.IsDeprecated);
```

Comment:
>.Where(f => f is not ISupportsDeprecatedFunctions sdf || !sdf.IsDeprecated) [](http://example.com/codeflow?start=90&length=75)

This is in common code with Power Apps - how is PA solving this problem?

Reply:
discussed offline: 
Power Apps does not yet do this. If it's deprecated, they completely ignore the method - so it's not callable. 

Here, it's still callable, but but just doesn't show in intellisense.